core/src/main/scala/kafka/server/ClientRequestQuotaManager.scala

```diff
@@ -30,6 +30,9 @@ import scala.jdk.CollectionConverters._
 object ClientRequestQuotaManager {
   val QuotaRequestPercentDefault = Int.MaxValue.toDouble
   val NanosToPercentagePerSecond = 100.0 / TimeUnit.SECONDS.toNanos(1)
+  // Since exemptSensor is for all clients and has a constant name, we do not expire exemptSensor and only
+  // create once.
+  val DefaultInactiveExemptSensorExpirationTimeSeconds = Long.MaxValue
```

Comment:
To clarify, I'm just suggesting we get rid of `DefaultInactiveExemptSensorExpirationTimeSeconds` and use `Long.MaxValue` directly. We already have the comment which emphasizes that the sensor does not get expired.

```scala
// Since exemptSensor is for all clients and has a constant name, we do not expire it
val exemptSensor: Sensor = getOrCreateSensor(
  sensorName = ClientRequestQuotaManager.ExemptSensorName, 
  expirationTimeMs = Long.MaxValue,
  metricRegistrationCallback = sensor => sensor.add(exemptMetricName, new Rate)
)
```

Reply:
I prefer using DefaultInactiveExemptSensorExpirationTimeSeconds because the value could be updated if needed. And it aligns with the code pattern, e.g., we also have QuotaRequestPercentDefault which has Int.MaxValue